kolibri_server_setup.py

```diff
@@ -161,64 +167,60 @@ def save_nginx_conf_port(port, nginx_conf=None):
     if nginx_conf is None:
         nginx_conf = os.path.join(KOLIBRI_HOME, "nginx.conf")
 
-    # If a port is already in the file, let's not add a new one:
-    with open(nginx_conf, "r") as nginx_conf_file:
-        data_configs = nginx_conf_file.read()
-    if "listen" in data_configs:
-        return
+    configuration = (
+        "# This file is maintained AUTOMATICALLY and will be overwritten\n"
+        "#\n"
+        "# Do not edit this file. If you are using the kolibri-server"
+        "package,\n"
+        "# please write custom configurations in /etc/kolibri/nginx.d/\n"
+        "\n"
+        "server{{\n"
+        "  listen {port};\n"
+        "  location {path_prefix}favicon.ico {{\n"
+        "    empty_gif;\n"
+        "  }}\n\n"
+        "  location {path_prefix} {{\n"
+        "    include uwsgi_params;\n"
+        "    uwsgi_pass unix:///tmp/kolibri_uwsgi.sock;\n"
+        "    proxy_ignore_headers Vary;\n"
```

Comment:
We should remove this, as otherwise it will cache inappropriately. I think this was originally added because Django by default adds Vary to everything because of the Session, but we are deliberately stripping that out of pages that can be cached, so their Vary header should be accurate.

Reply:
Done